presto-docs/src/main/sphinx/release/release-0.275.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.275
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator to correctly identify running queries as leaked.
+* Improve the memory limit error message with allocationTag.
+* Add :func:`trimmed_mean` for :ref:`tdigest <tdigest_type>`. The prototype of the function is: trimmed_mean(tdigest: TDIGEST, lower_quantile: DOUBLE, upper_quantile: DOUBLE) -> DOUBLE.
+* Add a router in front of presto clusters. The router is deployed as a standalone service with a presto-styled UI.
+* Add new fields in Iceberg files system table.
+* Add raft server to resource manager with disaggregated coordinators using Apache Ratis. Raft server enabled in resource manager using the ``isEnabled`` property in Config file.
```

Comment:
Few separate comments:
- Would be good to add a sentence to explain the use/impact of the new feature.
- Probably it's `raft.isEnabled`
- not related to release note but to the code : we don't usually use camel case for configs.... `isEnabled` should be `is-enabled` but that's fine for now.